eks-cluster.tf

```diff
@@ -96,7 +96,7 @@ module "eks" {
         xvda = {
           device_name = "/dev/xvda"
           ebs = {
-            volume_size           = 200 # Same size as DigitalOcean's size. TODO: Synchronize values between all cloud providers
+            volume_size           = 50
```

Comment:
do we only every run 1 pod per node at the same time?

or is this volume per pod?

Reply:
You're right, there might be multiple pods running at the same time on the same node, I don't know how much though.

Reply:
As it's for volumes mounted as `emptyDir`, all of them would compete for the available space on each node, AFAIK there is nothing in that case ensuring they'll get a reserved space.

Reply:
As we run 3 pods per nodes, that means we need at least 69 Gb for the emptyDir in current state.
If you had the system space (+ docker images and other emptyDirs), that means ~ 90 Gb is needed.